src/Integration.Vsix/Analysis/IssueConsumerFactory.cs

```diff
@@ -51,17 +52,19 @@ internal partial class IssueConsumerFactory : IIssueConsumerFactory
     {
         private readonly ISuppressedIssueMatcher suppressedIssueMatcher;
         private readonly IAnalysisIssueVisualizationConverter converter;
+        private readonly ILocalHotspotsStoreUpdater localHotspotsStore;
 
         [ImportingConstructor]
-        internal IssueConsumerFactory(ISuppressedIssueMatcher suppressedIssueMatcher, IAnalysisIssueVisualizationConverter converter)
+        internal IssueConsumerFactory(ISuppressedIssueMatcher suppressedIssueMatcher, IAnalysisIssueVisualizationConverter converter, ILocalHotspotsStoreUpdater localHotspotsStore)
```

Comment:
Changing the issue consumer to use the new store is logically separate and should have been a separate ticket and done in a separate PR. It's not worth separating the changes out for this PR, but going forwards please create separate tickets for each logically separate tasks.